src/Makefile

```diff
@@ -1,3 +1,6 @@
+# @file Makefile
+# @author Jan Rodak - xrodak00
+# @brief Makefile
```

Comment:
U doxygen komentářů dodržujeme konvenci 
""" 
@file bla bla
@author bla bla
atd...
"""
Přepiš to pls

Reply:
V Makefile to takto nefunguje ale našel jse [řešení](https://stackoverflow.com/questions/26874238/how-to-document-makefile-templates-and-include-mk-file-interfaces).